### PR TITLE
i18n(zh-cn): Update `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by Nin3lee**
#### Description (required)

Update `configuration-reference.mdx`

#### Related issues & labels (optional)

- #8842 
- Suggested label: `i18n`

### CODE REVIEW

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -1662,3 +1662,66 @@ export default defineConfig({
...
+const user = await getUser(Astro.locals.userId);
+---
+<img class="avatar" src={user.imageUrl}>
+```
+
+#### 服务器群岛的后备内容
+
+由于你的组件不会与页面的其他部分一起渲染，因此可能需要添加通用内容（例如：加载信息）以临时替代它。这些后备内容将在页面上群岛加载之前的首次渲染时显示。
+
+使用 `slot="fallback:` 属性将占位符内容添加为 Astro 组件的子组件。当群岛内容可用时，后备内容就会被替换。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1717]:
```suggestion
使用 `slot="fallback"` 属性将占位符内容添加为 Astro 组件的子组件。当群岛内容可用时，后备内容就会被替换。
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 2]
```diff
@@ -1662,3 +1662,66 @@ export default defineConfig({
...
+---
+import { getUser } from '../api';
+const user = await getUser(Astro.locals.userId);
+---
+<img class="avatar" src={user.imageUrl}>
+```
+
+#### 服务器群岛的后备内容
+
+由于你的组件不会与页面的其他部分一起渲染，因此可能需要添加通用内容（例如：加载信息）以临时替代它。这些后备内容将在页面上群岛加载之前的首次渲染时显示。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1715]:
```suggestion
由于你的组件不会与页面的其他部分一起渲染，因此可能需要添加通用内容（例如：加载中）以临时显示在该位置。这些内容将在页面上群岛加载之前的首次渲染时显示。
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 3]
```diff
@@ -1662,3 +1662,66 @@ export default defineConfig({
...
+
+使用 `slot="fallback:` 属性将占位符内容添加为 Astro 组件的子组件。当群岛内容可用时，后备内容就会被替换。
+
+下面的例子展示了将一个通用头像作为后备内容后，利用视图过渡动画转换为个性化头像的情况：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1719]:
```suggestion
下面的例子展示了将一个通用头像作为回退内容后，利用视图过渡动画转换为个性化头像的情况：
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 4]
```diff
@@ -1662,3 +1662,66 @@ export default defineConfig({
...
+
+```astro
+<Avatar server:defer>
+  <svg slot="fallback" class="generic-avatar" transition:name="avatar">...</svg>
+</Avatar>
+```
+
+如需了解完整概述，或欲对该实验性 API 提供反馈建议，请参阅 [服务器群岛 RFC](https://github.com/withastro/roadmap/pull/963)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1727]:
```suggestion
要全面了解并对这个实验性 API 提供反馈，请查看 [服务器群岛 RFC](https://github.com/withastro/roadmap/pull/963)。
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 5]
```diff
@@ -1662,3 +1662,66 @@ export default defineConfig({
...
+
+```astro
+---
+import { getUser } from '../api';
+const user = await getUser(Astro.locals.userId);
+---
+<img class="avatar" src={user.imageUrl}>
+```
+
+#### 服务器群岛的后备内容
```
[the remark this answers is not in the document]
Comment (Nin3lee) [reply src/content/docs/zh-cn/reference/configuration-reference.mdx]: fallback => 回退
学到了，感谢@liruifengv

**Review by liruifengv — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**